mne/viz/utils.py

```diff
@@ -1074,6 +1075,10 @@ def plot_sensors(
         ``kind='3d'``, or ``25`` otherwise.
     linewidth : float
         The width of the outline. If ``0``, the outline will not be drawn.
+    cmap : str | instance of matplotlib.colors.Colormap | None
+        Colormap for coloring ch_groups. Has effect only when ``ch_groups``
+        is list of list. If None, set to matplotlib.rcParams["image.cmap"].
```

Comment:
Code mode will probably look better here
```suggestion
        is list of list. If None, set to ``matplotlib.rcParams["image.cmap"]``.
```